### ISSUE TITLE
多卡场景pod内部nvidia-smi显存占用很低，node中显存占用正常

### ISSUE THREAD

**Issue by Awadabang** [Project-HAMi/HAMi#1180]
**What happened**: 

使用llamafactory进行训练时多卡场景pod内部nvidia-smi显存占用很低，node中显存占用正常；使用gpu_burn测试显存正常显示

**What you expected to happen**:

pod中nvidia-smi正常显示显存占用

**How to reproduce it (as minimally and precisely as possible)**:

pod分配2两个完整显卡，nvidia-smi正常

执行：`llamafactory-cli train examples/train_lora/qwen25_lora_sft.yaml`

> ```
> ### qwen25_lora_sft.yaml
> ### model
> model_name_or_path: /workspace/models/Qwen2.5-14B-Instruct
> #adapter_name_or_path: /workspace/tools/LLaMA-Factory/saves/qwen25-14b/lora/pt200-r32/checkpoint-1000
> trust_remote_code: true
> 
> ### method
> stage: sft
> do_train: true
> finetuning_type: lora
> lora_rank: 32
> lora_target: all
> 
> ### dataset
> dataset: merge_extract
> template: qwen
> cutoff_len: 4096
> max_samples: 100000
> overwrite_cache: true
> preprocessing_num_workers: 16
> dataloader_num_workers: 4
> 
> ### output
> output_dir: saves/qwen25-14b/lora/sft-r32-test
> logging_steps: 10
> save_steps: 200
> plot_loss: true
> overwrite_output_dir: true
> save_only_model: false
> report_to: none  # choices: [none, wandb, tensorboard, swanlab, mlflow]
> 
> ### train
> per_device_train_batch_size: 1
> gradient_accumulation_steps: 16
> learning_rate: 5.0e-5
> num_train_epochs: 100.0
> lr_scheduler_type: cosine
> warmup_ratio: 0.05
> bf16: true
> ddp_timeout: 180000000
> #resume_from_checkpoint: /workspace/tools/LLaMA-Factory/saves/qwen25-14b/lora/sft-r32/checkpoint-13600
> use_dora: true
> 
> ### eval
> #eval_dataset: alpaca_en_demo
> #val_size: 0.1
> #per_device_eval_batch_size: 1
> #eval_strategy: steps
> #eval_steps: 50```

pod内部显示两张卡分别只用了400MB显存
node显示两张卡用了30GB显存

发现了这两行日志：
[HAMI-core Msg(10978:140261672473664:memory.c:512)]: orig free=46993309696 total=47677177856 limit=48297410560 usage=444596224
[HAMI-core Msg(10979:140718493594688:memory.c:512)]: orig free=46993309696 total=47677177856 limit=48297410560 usage=444596224

**Anything else we need to know?**:

- The output of `nvidia-smi -a` on your host
- Your docker or containerd configuration file (e.g: `/etc/docker/daemon.json`)
- The hami-device-plugin container logs
- The hami-scheduler container logs
- The kubelet logs on the node (e.g: `sudo journalctl -r -u kubelet`)
- Any relevant kernel output lines from `dmesg`

**Environment**:
- HAMi version: 
- nvidia driver or other AI device driver version: Nvidia Driver Version: 570.158.01
- Docker version from `docker version`
- Docker command, image and tag used
- Kernel version from `uname -a`：Linux yai-prod-k8s-gpu-node-07 5.14.0-570.23.1.el9_6.x86_64 #1 SMP PREEMPT_DYNAMIC Tue Jun 24 11:27:16 EDT 2025 x86_64 x86_64 x86_64 GNU/Linux
- Others:

**Comment by mesutoezdil** [comment]
Hi,

We are doing a cleanup of open issues to keep the backlog manageable. This issue was opened before 2026 and has not seen recent activity.

Please reply by 2026-07-13 to confirm this is still relevant and reproducible on the current version of HAMi. If we do not hear back, we will close this issue. You can always reopen it or file a new one if the problem persists.

Thank you for contributing to HAMi.
